app/policies/article_policy.rb

```diff
@@ -20,6 +20,10 @@ def admin_unpublish?
     user_any_admin?
   end
 
+  def admin_featured_toggle?
```

Comment:
I think recently @jeremyf started moving towards aliasing instead of explicitly wrapping policies, maybe double check with him as he's doing most of the current authorization work.